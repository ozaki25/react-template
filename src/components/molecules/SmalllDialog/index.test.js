import React from 'react';
import { snapshot } from 'test/helpers';
import SmallDialog from '.';

const props1 = {
  title: 'タイトル',
  children: 'コンテンツ',
  buttonLabel: 'OK',
  onClick: jest.fn(),
};

const props2 = {
  children: 'コンテンツ',
  buttonLabel: 'OK',
  onClick: jest.fn(),
};

const props3 = {
  children: 'コンテンツ',
  buttonLabel: 'OK',
  onClick: jest.fn(),
  show: false,
};

snapshot('SmallDialog/タイトルあり', <SmallDialog {...props1} />);
snapshot('SmallDialog/タイトルなし', <SmallDialog {...props2} />);
snapshot('SmallDialog/非表示', <SmallDialog {...props3} />);
