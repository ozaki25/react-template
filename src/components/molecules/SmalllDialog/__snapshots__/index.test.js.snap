// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match snapshot: SmallDialog/タイトルあり 1`] = `
.c6 {
  background-color: #333;
  border-radius: 4px;
  color: #fff;
  font-size: 16px;
  font-weight: bold;
  height: 44px;
  width: 150px;
}

.c2 {
  color: #333;
  font-size: 17px;
  font-weight: bold;
}

.c4 {
  color: #333;
  font-size: 16px;
  font-weight: nomal;
}

.c0 {
  background-color: #f5f5f5;
  border-radius: 4px;
  padding: 10px;
  text-align: center;
}

.c1 {
  margin-bottom: 5px;
  margin-top: 5px;
}

.c3 {
  margin-bottom: 5px;
  margin-top: 5px;
}

.c5 {
  margin-bottom: 5px;
  margin-top: 5px;
}

<SmallDialog
  buttonLabel="OK"
  onClick={[Function]}
  title="タイトル"
>
  <styled.div>
    <div
      className="c0"
    >
      <styled.div>
        <div
          className="c1"
        >
          <BasicText
            emphasis={true}
            red={false}
          >
            <styled.p
              emphasis={true}
              red={false}
            >
              <p
                className="c2"
              >
                タイトル
              </p>
            </styled.p>
          </BasicText>
        </div>
      </styled.div>
      <styled.div>
        <div
          className="c3"
        >
          <BasicText
            emphasis={false}
            red={false}
          >
            <styled.p
              emphasis={false}
              red={false}
            >
              <p
                className="c4"
              >
                コンテンツ
              </p>
            </styled.p>
          </BasicText>
        </div>
      </styled.div>
      <styled.div>
        <div
          className="c5"
        >
          <BasicButton
            disabled={false}
            onClick={[Function]}
            red={false}
            small={false}
            xsmall={true}
          >
            <styled.button
              disabled={false}
              onClick={[Function]}
              red={false}
              small={false}
              xsmall={true}
            >
              <button
                className="c6"
                disabled={false}
                onClick={[Function]}
              >
                OK
              </button>
            </styled.button>
          </BasicButton>
        </div>
      </styled.div>
    </div>
  </styled.div>
</SmallDialog>
`;

exports[`should match snapshot: SmallDialog/タイトルなし 1`] = `
.c4 {
  background-color: #333;
  border-radius: 4px;
  color: #fff;
  font-size: 16px;
  font-weight: bold;
  height: 44px;
  width: 150px;
}

.c2 {
  color: #333;
  font-size: 16px;
  font-weight: nomal;
}

.c0 {
  background-color: #f5f5f5;
  border-radius: 4px;
  padding: 10px;
  text-align: center;
}

.c1 {
  margin-bottom: 5px;
  margin-top: 5px;
}

.c3 {
  margin-bottom: 5px;
  margin-top: 5px;
}

<SmallDialog
  buttonLabel="OK"
  onClick={[Function]}
  title=""
>
  <styled.div>
    <div
      className="c0"
    >
      <styled.div>
        <div
          className="c1"
        >
          <BasicText
            emphasis={false}
            red={false}
          >
            <styled.p
              emphasis={false}
              red={false}
            >
              <p
                className="c2"
              >
                コンテンツ
              </p>
            </styled.p>
          </BasicText>
        </div>
      </styled.div>
      <styled.div>
        <div
          className="c3"
        >
          <BasicButton
            disabled={false}
            onClick={[Function]}
            red={false}
            small={false}
            xsmall={true}
          >
            <styled.button
              disabled={false}
              onClick={[Function]}
              red={false}
              small={false}
              xsmall={true}
            >
              <button
                className="c4"
                disabled={false}
                onClick={[Function]}
              >
                OK
              </button>
            </styled.button>
          </BasicButton>
        </div>
      </styled.div>
    </div>
  </styled.div>
</SmallDialog>
`;
